stm32-data-gen/src/low_power.rs

```diff
@@ -0,0 +1,62 @@
+use stm32_data_serde::chip::core::peripheral::rcc::StopMode;
+
+use crate::util::RegexMap;
+
+/// Get the stop mode limit for a peripheral based on the MCU and peripheral name.
+/// Determines the lowest possible stop mode when a peripheral is enabled.
+///
+/// Parameters:
+/// - mcu_name: the full name of the MCU (e.g., "STM32WB55RG")
+/// - peripheral: the name of the peripheral (e.g., "USART1")
+pub(crate) fn get_peripheral_stop_mode_override(mcu_name: &str, peripheral: &str) -> Option<StopMode> {
+    /// Regexmap where the key is mcu_name:peripheral and the value is the stop mode.
+    /// Example: STM32WB55RG:USART1 -> StopMode::Stop2
+    #[rustfmt::skip]
+    static STOP_MODE_OVERRIDE_RULES: RegexMap<StopMode> = RegexMap::new(&[
+        (r"^STM32WB55.*:LPTIM1", StopMode::Standby),
+        (r"^STM32WB55.*:USART1", StopMode::Stop2),
+        (r"^STM32WB55.*:LPUART1", StopMode::Standby),
```

Comment:
Are you sure this is correct? I don't think LPUARTs can work in standby.

Reply:
The can work in STOP2.
The StopMode enum limits it up to the certain level.
That's the enum. I just changed the comments to doc comments
``` rust
  pub enum StopMode {
      #[default]
      /// Peripheral prevents chip from entering Stop1
      Stop1,
      /// Peripheral prevents chip from entering Stop2
      Stop2,
      /// Peripheral does not prevent chip from entering Stop
      Standby,
  }
```

Reply:
But maybe we could adjust the naming of the enum to be more intuitive

Reply:
oh wow yes this is somewhat unintuitive. Intuitively i'd say "Standby" means "works in Standby", not "works in all modes up to but not including Standby"

Reply:
Yeah. But I am not sure if the name can be changed easily because of the metapac gen and so on